api/nodes/views.py

```diff
@@ -1652,7 +1653,14 @@ def get_queryset(self):
 
     # overrides ListCreateAPIView
     def perform_create(self, serializer):
-        serializer.save(node=self.get_node())
+        user = get_user_auth(self.request).user
+        node = self.get_node()
+        try:
+            fork = serializer.save(node=node)
+            mails.send_mail(user.email, mails.FORK_COMPLETED, title=fork.title, guid=fork._id, mimetype='html')
+        except Exception, exc:
```

Comment:
Very minor; the standard pattern used in the osf is except <exception> as <error>